roles/ldap/tasks/main.yml

```diff
@@ -69,6 +69,13 @@
     - "dc eq"
     - "entryCSN eq"
 
+- name: Set olcDatabase={-1}frontend olcSizeLimit
+  community.general.ldap_attrs:
+    dn: "olcDatabase={-1}frontend,cn=config"
+    state: "exact"
+    attributes:
+      olcSizeLimit: "unlimited"
+
```

Comment:
Dit lijkt me niet in deze PR thuishoren?
Hebben we deze change nog wel nodig?  Ik dacht dat we dit geneuzel na de zomer helemaal gefixed hadden.

Reply:
blijkbaar wel echt nodig, als ik de commits zo zie?
Kan ook geen kwaad verder.